src/lib/types/CommonInputProps.ts

```diff
@@ -8,6 +8,7 @@ interface CommonInputProps<T extends FieldValues> {
   name: keyof T;
   id?: string;
   helpText?: ReactNode;
+  disabled?: boolean;
```

Comment:
`CommonInputProps` is used on all the different inputs. 

Could you therefor please make sure that all the inputs make use of it?

Reply:
Updated all other inputs and stories